src/components/App.js

```diff
@@ -19,4 +20,8 @@ function App() {
   );
 }
 
+App.propTypes = {
+  monster: PropTypes.object.isRequired,
```

Comment:
This is just an example of how propTypes are written out - we'll make it actually correct when we figure out which props this component is pulling in!